geode-core/src/integrationTest/java/org/apache/geode/internal/JarDeployerIntegrationTest.java

```diff
@@ -168,7 +169,12 @@ public void undeploy() throws Exception {
         .isInstanceOf(IllegalArgumentException.class);
```

Comment:
It's not clear in the test what makes this argument illegal. I would prefer to put this in a separate test, and give the test a name that describes why the argument is illegal.